Nos/Views/Discover/FeaturedAuthorsView.swift

```diff
@@ -4,7 +4,8 @@ import Dependencies
 
 struct FeaturedAuthorsView: View {
     @EnvironmentObject private var router: Router
-    
+    @Environment(\.managedObjectContext) private var context
```

Comment:
Can you rename context to viewContext so we have the same variable name for that context in all SwiftUI views?

Reply:
@martindsq sure. I'm wondering if we should be using PersistenceController here instead of a managedObjectContext. I don't know enough about our architecture to know what the right solution is here (especially to avoid deadlock which I think is happening [here](https://github.com/planetary-social/nos/issues/970)). Do you know if PersistenceController would be better here?

Reply:
Switched to persistenceController.